### PR TITLE
added announcement for 10.13 fix release

### PR DESCRIPTION
**Pull request by amsosag**
added announcement for 10.13 fix release

### CODE REVIEW

File: content/release-10-13-0/announcements-10-13-0.md [hunk 1]
```diff
@@ -171,3 +171,55 @@ The following applies as of Apama 10.11.1:
   This is because setting a small `pageSize` without setting `currentPage` can result in queries that run very slowly.
   For example, to request the first 20 items, you must set `pageSize` to 20 and `currentPage` to 1.
   See also [Paging Cumulocity IoT queries](https://documentation.softwareag.com/apama/v10-11/apama10-11/apama-webhelp/index.html#page/apama-webhelp%2Fco-ConApaAppToExtCom_cumulocity_paging_cumulocity_queries.html) in the Apama documentation.
+
+
+### Machine learning Workbench
```
Comment (BeateRixen) [inline content/release-10-13-0/announcements-10-13-0.md:176]:
```suggestion
### Machine Learning Workbench
```

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md [hunk 2]
```diff
@@ -60,3 +60,8 @@ The Notebook UI of Machine Learning Workbench was a custom implementation create
   - Fixed the issue with training more epochs on the integrated Jupyter Notebook within the MLW by upgrading tensorflow to 2.0.
   - Fixed the issue with committing the project when an existing project commit task is in-progress.
   - Fixed the issue with deleting the project when any of the project related task is in-progress.
+
+
+* Fix release:
+  - A potential security issue was detected in MLW microservice. To mitigate this, a fix release for MLW 10.13 is provided.
```
Comment (BeateRixen) [inline content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md:66]:
```suggestion
  - A potential security issue was detected in the MLW microservice. To mitigate this, a fix release for MLW 10.13 is provided.

```

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md [hunk 3]
```diff
@@ -60,3 +60,8 @@ The Notebook UI of Machine Learning Workbench was a custom implementation create
   - Fixed the issue with training more epochs on the integrated Jupyter Notebook within the MLW by upgrading tensorflow to 2.0.
   - Fixed the issue with committing the project when an existing project commit task is in-progress.
   - Fixed the issue with deleting the project when any of the project related task is in-progress.
+
+
+* Fix release:
+  - A potential security issue was detected in MLW microservice. To mitigate this, a fix release for MLW 10.13 is provided.
+    This fix release limits the access of "Machine Learning User" and "Machine Learning Manager" user groups. Detailed changes are documented here: [10.13 fix release](/release-10-13-0/announcements-10-13-0/#:~:text=the%20Apama%20documentation.-,Machine%20learning%20Workbench,-Implemented)
```
Comment (BeateRixen) [inline content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md:67]: Detailed changes are documented here: [10.13 fix release] > Detailed changes are documented under [Important announcements].
The link doesn´t look correct, should just be /release-10-13-0/announcements-10-13-0/

**Review by VRAN016 — APPROVED** [review]

File: content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md (hunk 3 above)
Comment (amsosag) [reply content/release-10-13-0/machine-learning-10-13-0-bundle/10_13_0.md]: I have updated the link

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**